go/vt/worker/executor.go

```diff
@@ -41,7 +41,7 @@ import (
 // executor is also used for executing vreplication and RefreshState commands.
 type executor struct {
 	wr        *wrangler.Wrangler
-	tsc       *discovery.LegacyTabletStatsCache
+	hc        *discovery.HealthCheckImpl
```

Comment:
Couldn't we use the interface instead of the impl?

Reply:
we definitely can! good catch

Reply:
this file has been removed on main